beacon-chain/cache/sync_committee.go

```diff
@@ -0,0 +1,187 @@
+// +build !libfuzzer
+
+package cache
+
+import (
+	"sync"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+	types "github.com/prysmaticlabs/eth2-types"
+	"github.com/prysmaticlabs/prysm/beacon-chain/state"
+	"github.com/prysmaticlabs/prysm/shared/bytesutil"
+	"k8s.io/client-go/tools/cache"
+)
+
+var (
+	maxSyncCommitteeSize = uint64(3) // Allows 3 forks to happen around `EPOCHS_PER_SYNC_COMMITTEE_PERIOD` boundary.
```

Comment:
Is this better to track in the global config? Might be hard to find in the future as it has no mention of "forks", it just says maxSyncCommitteeSize

Reply:
No, it's a cache specific value. I dont think we would put cache max size in global config. The comment means the max tolerance is 3 forks around sync committee period as that's where cache gets refreshed